### ISSUE TITLE
Could it be used remotely?

### ISSUE THREAD

**Issue by cbertelli** [stuporglue/ssmap#1]
Hello,
Could it be possible to use it remotely, say doing something like this:
<script src='http://from.remote.si.te/js/ssmap.js?div=floatydiv&amp;lat=-90&amp;lon=45'></script>
overcoming cross site limitations?
It would be very useful.

**Comment by stuporglue** [comment]
Yes, you might need to enable CORS (https://enable-cors.org/) on your
server, but in the end it just sends an image, so it should work just fine
remotely.

--
Michael Moore

**Comment by cbertelli** [comment]
Yes it does. Thanks
